fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dto/Resource.java

```diff
@@ -14,16 +14,47 @@
  * This class defines the Data Transfer Object representing a row in the FHIR Resource table.
  */
 public class Resource {
-
+    
+    /**
+     *  This is the <resourceType>_RESOURCES.RESOURCE_ID column
+     */
     private long id;
+    
+    /**
+     *  This is the <resourceType>_RESOURCES.LOGICAL_RESOURCE_ID column. It is only
```

Comment:
Should be `<resourceType>_LOGICAL_RESOURCES.LOGICAL_RESOURCE_ID` which is where it is the primary key. In the xx_RESOURCES table it is a foreign key. But in most queries, it will be read from the LR table.

Reply:
Done.